resources/assets/sass/components/user/_user-destination.scss

```diff
@@ -11,19 +21,79 @@
         margin-left: -4px;
     }
 
-    &:last-child:after {
+    &.m-box:after, &:last-child:after {
 
         content: none;
+        margin-left: 0;
+    }
+
+    &.m-box {
+
+        display:block;
+        float:left;
+    }
+
+    &.m-box:last-child > .c-user-destination__item-link {
+
+      margin-right: 0;
     }
 }
 
     .c-user-destination__item-link {
 
-        color: $color-fill-black-dark;
+        color: $color-text-link;
         display: inline-block;
+        font-size: $font-size-medium;
+        line-height: $font-size-medium * 1.25;
 
         &.m-white {
 
             color: $color-white;
         }
+
+        &.m-black {
+
+            color: $color-fill-black-dark;
+        }
+
+        .m-box & {
+
+            float: left;
+            display: block;
+            margin-bottom: $spacer / 1.5;
+            margin-right: $spacer / 1.5;
+        }
     }
+
+        .c-user-destination__item-text {
+
+            display: inline-block;
+            padding: $spacer / 2;
+            background: $color-fill-error-light;
+            box-shadow:inset 0px 0px 0px 1px $brand-red;
```

Comment:
Colon after property should be followed by one space<br>`0px` should be written without units as `0`